index.html

```diff
@@ -3,14 +3,62 @@
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Instagram Clone</title>
-    <!-- don't forget to link your styles -->
+    <link rel="stylesheet" href="./styles/style.css" />
   </head>
   <body>
-    Instagram Capture and Share the World's Moments Instagram is a fast, beautiful and fun way to share your life with
-    friends and family. Take a picture or video, choose a filter to transform its look and feel, then post to Instagram
-    &mdash; it's that easy. You can even share to Facebook, Twitter, Tumblr and more. It's a new way to see the world.
-    Oh yeah, did we mention it's free? Download on the App Store Get it on Google Play About Us Support Blog Press API
-    Jobs Privacy Terms &copy; 2014 Instagram
+    <div class="phones">
+      <img
+        id="phones-img"
+        src="../lab-css-instagram-clone/images/phones.png"
```

Comment:
No se si aca utilizas live-server o alguna herramienta similar, pero esta forma de importar los archivos no te va a funcionar.
Lo que tienes que hacer es usar una ruta relativa a donde están tus imagenes respecto al html sin `./`.

```suggestion
        src="images/phones.png"
```

Reply:
Así para todas tus imagenes.